rpmalloc/malloc.c

```diff
@@ -199,7 +199,7 @@ pthread_create(pthread_t* thread,
                const pthread_attr_t* attr,
                void* (*start_routine)(void*),
                void* arg) {
-#if defined(__linux__) || defined(__APPLE__)
+#if defined(__linux__) || defined(_FreeBSD__) || defined(__APPLE__)
```

Comment:
Should this be "defined(__FreeBSD__)" with a double underscore

Reply:
Yes a typo of mine ...